CHANGELOG.md

```diff
@@ -6,8 +6,15 @@ Changelog
 
 ### Announcements
 
-- Python 3.6 is no longer supported. Python 3.7, 3.8, and 3.9 are supported.
+- pyQuil now directly supports the QCS API v1.0, offering you better performance and more
+  granular data about QCS quantum processors.
 
+- Python 3.6 is no longer supported. Python 3.7, 3.8, and 3.9 are supported.
+  
+- `pyquil.compatibility.v2` provides a number of classes which support the pyQuil v2 API, such as
```

Comment:
Since many of the bullets in this release's changelog were written when the only API was the newer one, just be sure to go through all of them to make sure that the comments still apply correctly. For instance, if we had previously made a change that applied to `api.Something` update the comment to refer to `compatibility.v2.api.Something` instead, if the change was kept there, or remove the comment altogether if the change was reverted in `compatibility.v2.api.Something`. Hope that makes sense.

TL;DR: Make sure all the bullets that existed prior to this PR still make sense.

Reply:
definitely important - just gave it another look and I think it's good.